### DISCUSSION TITLE
2.4 Elastic Integration MISP Threat Intel

### DISCUSSION THREAD

**Discussion by 8inaryMata1eao** [Security-Onion-Solutions/securityonion#12253] [Unsupported Versions]
I am a long time user of SO, at least 10 year now. We are redeploying all of our client instances to 2.4.  Many of them are using MISP and have Graylog also capturing Syslog with zeek data.  Which is wasteful because we have the same data in two places for the benefits of both platforms.  In this instance of SO I am really loving the elastic agent integrations with sysmon and others.  We run a MISP instance to share data across all of our client base.  In the elastic agent there is a MISP integration.  It is setup to pull the data but all it seems to do it just dump the data into elasticsearch.  Is there a process to get this integration to provide enrichment?  Or is it still the previous method using the [threat intel module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html)?  If so, what is the value of the elastic agent MISP integration?

---------------------------------------------------------------------------------------------------------
Wes had provided this a few years ago, does it still translate to this implementation?
https://glue.ghost.io/leveraging-threat-intel-for-event-enrichment-in-security-onion/

**Comment by reyesj2** [comment]
You still need to setup an enrich policy + enrich pipeline. That blog post should still be helpful for setting up the enrich policy.
